pipeline.py

```diff
@@ -112,32 +91,32 @@ def pipeline(
             int
         ] = None,  # FIXME: https://issues.redhat.com/browse/RHOAIRFE-467
         sdg_base_model: str = "s3://<BUCKET>/<PATH_TO_MODEL>",
-        sdg_scale_factor: int = 2,  # Renamed upstream https://github.com/instructlab/instructlab/blob/f7d40f6ed5112d59132dd832bd332fa6fbbe7010/src/instructlab/configuration.py#L279-L290
-        sdg_pipeline: str = SDG_PIPELINE,
-        sdg_max_batch_len: int = MAX_BATCH_LEN,
-        sdg_sample_size: float = 1.0,
+        sdg_scale_factor: int = 30,  # https://github.com/instructlab/instructlab/blob/d68c5a12026e08092bd4171a77667b09f7c36f72/tests/testdata/default_config.yaml#L125
+        sdg_pipeline: str = "full",  # https://github.com/instructlab/instructlab/blob/d68c5a12026e08092bd4171a77667b09f7c36f72/tests/testdata/default_config.yaml#L122
+        sdg_max_batch_len: int = 5000,  # https://github.com/instructlab/instructlab/blob/d68c5a12026e08092bd4171a77667b09f7c36f72/tests/testdata/default_config.yaml#L334
+        sdg_sample_size: float = 1.0,  # FIXME: Not present in default config. Need justification?
         # Training phase
-        train_nproc_per_node: int = 3,
-        train_nnodes: int = 2,
-        train_num_epochs_phase_1: int = NUM_EPOCHS_PHASE_1,
-        train_num_epochs_phase_2: int = NUM_EPOCHS_PHASE_2,
-        train_effective_batch_size_phase_1: int = EFFECTIVE_BATCH_SIZE_PHASE_1,
-        train_effective_batch_size_phase_2: int = EFFECTIVE_BATCH_SIZE_PHASE_2,
-        train_learning_rate_phase_1: float = LEARNING_RATE_PHASE_1,
-        train_learning_rate_phase_2: float = LEARNING_RATE_PHASE_2,
-        train_num_warmup_steps_phase_1: int = NUM_WARMUP_STEPS_PHASE_1,
-        train_num_warmup_steps_phase_2: int = NUM_WARMUP_STEPS_PHASE_2,
-        train_save_samples: int = SAVE_SAMPLES,
-        train_max_batch_len: int = MAX_BATCH_LEN,
-        train_seed: int = SEED,
+        train_nproc_per_node: int = 3,  # FIXME: Not present in default config. Need justification?
+        train_nnodes: int = 2,  # FIXME: Not present in default config. Need justification?
```

Comment:
ilab does not do any distributed training across nodes so this is normal such flag does not exist.

Reply:
The question is, do we have a justification for the defaults we've selected here? Do we need it? Is 2*3 the best default config we can produce? 🙂

Reply:
I think we should either do 2*1 or 2*2 as default.  

2*1 would be the absolute minimum for mulit-pod distribution.
2*2 would be the minimum for multi-gpu, multi-pod distribution.

I vote for 2*2 so the default is showing the full capabilities of the  system, but happy to hear other opinions.

Reply:
Ok.. so what if I leave those `FIXME` in place for the time being and explain that we're waiting for a reference architecture? WDYT? Would that make sense?

Reply:
I can combine that with what @MichaelClifford says above.. change that 2x2 + add a note the RHOAI team needs to provide their justification or change it. 🙂

Reply:
Updated both to be 2x2 and added a note. WDYT now? 🙂